main.yml

```diff
@@ -161,6 +161,20 @@ landing:
   channel: '#flip-mode-squad'
   deployment_repo: https://github.com/RedHatInsights/landing-page-frontend-build
 
+migrations:
+  title: Migrations
+  api: 
+    versions:
+      - v1
+  frontend: 
+    title: Migrations
+    sub_apps:
+      - id: migration-analytics
+        title: Migration Analytics
```

Comment:
Supplying a `title` is only necessary if the sub-app isn't defined elsewhere in the config; this field is for things like `/insights/overview`, which aren't a separate app and just act as routing for `/insights`.

Reply:
Removed